docs: add growing your project section (#291)
open-sauced/docs: docs/maintainers/growing-your-project.md

- Identify active contributors in other repositories who may be interested in joining your team.

If you need more tips on how to pick people that can help grow your project, check out our blog post, ["How to Build Your Open Source Dream Team"](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90).

## Recruiting Team Members and Contributors Outside of Your Project 

When you're ready to recruit your team members, consider creating a [Repository Insight Page](../features/repo-insights.md) with projects that are similar to yours. From there, determine which contributors suit your project, add them to a list, and monitor their progress.  

:::tip

Consider checking out our [Understanding Repository Insights Data](../features/repo-insights.md) guide to give you an idea on what to look for when narrowing down your list.

:::

Once you've picked the people you want to join your project, recruit them. You can do this by going to their [OpenSauced profiles](../contributors/contributors-guide.md#creating-your-profile) and sending them a message via the contact information they provide.

## Understanding team participation

> Once you have recruited your official team members, consider adding them to your project's Contributors Insights page by using [the team sync feature](../features/contributor-insights.md#how-to-create-a-contributor-insight-page). This can be useful in monitoring their progress.

Author: ```suggestion
Once you've identified the people you want to ask, reach out to them by providing a clear message and a list of expectations.
```

> 

Author: [question/suggestion]: Would it be fitting to mention the Pitch Your Projects X space as a tip?

Reviewer: I'm on my mobile atm and might not be able to review this today. But can you let me know to which direction you go?

Author: Hi @adiati98. This issue was raised months ago so I based this information on the recent team workspace feature and my own experience being recruited to being a maintainer at LinksHub.

Reviewer: @CBID2, sorry, I didn't see that you've commented already when I edited my previous comment. 😅 
I've added an explanation about the repetitive topic that you've mentioned that you might need to read.

I have a question, though. Which recent team workspace feature do you mean?

I might be wrong. But I think what Bekah wants for this section is mainly about recruiting new team members with OpenSauced. Maintainers can use insights features as a tool to monitor contributors from other projects that have the potential to be team members. 

If this is the case, I don't see why the suggestions are pretty outdated. 🙂

That being said, I'm not sure if your suggestions fit in here. But I'll let Bekah decide.

Reviewer 2: @CBID2 we shouldn't mention anything in the docs for events, unless it's #100Daysofoss. It's more to maintain. 

If you're thinking about going in a direction that differs from the issue, please always ask first. I think it's a good idea to talk about workspaces, but this should focus on insights.

If you have a workspace for your project(s), you can create contrib and repo insights to help you grow your project in that workspace.

Author: Hi @BekahHW. I made the updates! :)

---
id: growing-your-project
title: "Growing Your Project"
sidebar_label: "Growing Your Project"
keywords:
  - "maintainers"

>   - "guides"

Reviewer 2: Let's add more key words here:
"open source maintainers" "growing an open source project" "open source project success" "maintainers guide"

---

## Growing your project

Growth is the the crucial aspect of a successful open source project. If you're at this stage in your open source maintainer journey, here are some things to consider when thinking of growth strategies.

### When to grow your project

> When it comes to deciding on whether your open source project is ready to grow, consider reviewing your project's Repository Insights page and asking yourself the following questions:

Reviewer 2: Link to docs for repo insight page. It also might be useful to have 

:::info
Repository Insights pages give an overview of open source projects, including more information on...

:::

- Are there tasks that are becoming too overwhelming for me and/or team?
- How can I improve and maintaining my project's current and pending features?
- How many stars and forks do I want to reach this time, and who can help me achieve it?
If you have said yes to these questions, it's time to develop your growth strategies. If you haven't already, consider creating different teams for specific purposes. For example, you can create a docs team for documentation maintenance, a triage team for reviewing and approving issues, or a maintainer team to help you with overall oversight of the project.  Once you have decided the types of teams you want to add to your project, it's time to start to building the open source team of your dreams.

### How to identify new contributors and team members

> When it comes to planning the growth strategies for your open source project, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important.  When doing this process, consider going to the Contributors tab in your project's Repository Insights page. While you do this, ask yourself the following questions:

Reviewer: ```suggestion
When it comes to planning the growth strategies for your open source project, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important. When doing this process, consider going to the "Contributors" tab in your project's Repository Insight Page. While you do this, ask yourself the following questions:
```

- How often do they contribute to yours and other open source projects in general?
- Are they invested in your project's success?

> Once you have made your observations, [create an a contributor insights page for your project](../maintainers/maintainers-guide.md#creating-a-new-contributor-insight-page). This will help you to make decisions about what steps you want to take next. Here are some approaches you can use to help you:

Reviewer: There's a grammar error and need capitalization here.

```suggestion
Once you have made your observations, [create a Contributor Insight Page for your project](../maintainers/maintainers-guide.md#creating-a-new-contributor-insight-page). This will help you to make decisions about what steps you want to take next. Here are some approaches you can use to help you:
```

- Monitor contributors to see if they are active enough in your repositories to contribute as a team member.
- Identify your top contributors to extend an invitation to join a team.
- Identify active contributors in other repositories who may be interested in joining your team.

> If you need more tips on how to pick people that can help grow your project, check out our blog post, ["How to Build Your Open Source Dream Team"](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90).

Reviewer: @BekahHW I have a general question here. Do we want to mention blog posts in our docs? Because I checked, so far, we don't mention any blog post in our docs.

Reviewer 2: @adiati98 I think it's ok to link it in the guides, just not in the feature documentation. But I will say if there are options to link to another part of the docs, the courses, or a blog post, this is the order of prioritization:

1. docs
2. courses
3. blog posts

---
id: growing-your-project
title: "Growing Your Project"
sidebar_label: "Growing Your Project"
keywords:
  - "maintainers"
  - "guides"
---

## Growing your project

Growth is the the crucial aspect of a successful open source project. If you're at this stage in your open source maintainer journey, here are some things to consider when thinking of growth strategies.

### When to grow your project

> When it comes to deciding on whether your open source project is ready to grow, consider reviewing your project's repository insights page in its Workspace page and asking yourself the following questions:

Reviewer 2: Let's capitalize Repository and Contributor Insights page.

- Are there tasks that are becoming too overwhelming for me and/or team?
- How can I improve and maintaining my project's current and pending features?
- How many stars and forks do I want to reach this time, and who can help me achieve it?
If you have said yes to these questions, it's time to develop your growth strategies. If you haven't already, consider creating different teams for specific purposes. For example, you can create a docs team for documentation maintenance, a triage team for reviewing and approving issues, or a maintainer team to help you with overall oversight of the project.  Once you have decided the types of teams you want to add to your project, it's time to start to building the open source team of your dreams.

### How to identify new contributors and team members

> 

Reviewer 2: I don't think we need to mention devcards or even profiles here. The best way to get a quick overview is with a Contributor Insight page or by creating a Repo Insight page, looking at the contributors there and then adding them to a Contrib Insight page.

:::

After you finished your observations, consider narrowing down your list to the contributors that suit your criteria the most. If you need more tips on how to pick people that can help grow your project, check out our blog post, ["How to Build Your Open Source Dream Team"](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90).

### Recruiting team members and contributors

When you're ready to recruit your team members, consider creating an [repository insights page](../docs/features/repo-insights.md) with projects that are similar to yours. From there, determine which contributors you think suits your project, add them to a list, and monitor their progress.  

:::tip
Consider checking out our [Understanding Repository Insights Data](../docs/features/understanding-repo-insights.md) guide to give you an idea on what to look for when narrowing down your list.
:::

Once you've finished your observations, recruit them. You can do this by going to their OpenSauced profiles and sending them a message via the contact information they provide.

### Understanding team participation

> Once you have picked your official team members, it is crucial to monitor their participation to ensure that they are help the project run efficiently.  You can do this by using [the team sync feature](../docs/features/contributor-insights.md#how-to-create-a-contributor-insight-page).

Reviewer 2: I wouldn't use crucial here. It can be "useful" or "helpful" to monitor your teams activity level by using a Contributor Insight page.

- How often do they contribute to yours and other open source projects in general?
- Are they invested in your project's success?

Once you have made your observations, [create an a contributor insights page for your project](../maintainers/maintainers-guide.md#creating-a-new-contributor-insight-page). This will help you to make decisions about what steps you want to take next. Here are some approaches you can use to help you:

- Monitor contributors to see if they are active enough in your repositories to contribute as a team member.
- Identify your top contributors to extend an invitation to join a team.
- Identify active contributors in other repositories who may be interested in joining your team.

If you need more tips on how to pick people that can help grow your project, check out our blog post, ["How to Build Your Open Source Dream Team"](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90).

### Recruiting team members and contributors

> When you're ready to recruit your team members, consider creating an [repository insights page](../features/repo-insights.md) with projects that are similar to yours. From there, determine which contributors you think suits your project, add them to a list, and monitor their progress.  

Reviewer: There are grammar errors. Also, please keep consistency for the capitalization.

```suggestion
When you're ready to recruit your team members, consider creating a [Repository Insight Page](../features/repo-insights.md) with projects that are similar to yours. From there, determine which contributors suit your project, add them to a list, and monitor their progress.  
```

> :::tip
> Consider checking out our [Understanding Repository Insights Data](../features/repo-insights.md) guide to give you an idea on what to look for when narrowing down your list.
> :::

Reviewer: Add a space before and after the content of the tip to [avoid problem with Prettier](https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier) if we have one in the future.

```suggestion
:::tip

Consider checking out our [Understanding Repository Insights Data](../features/repo-insights.md) guide to give you an idea on what to look for when narrowing down your list.

:::
```

---
id: growing-your-project
title: "Growing Your Project"
sidebar_label: "Growing Your Project"
keywords:
  - "open source maintainers"
  - "maintainers guides"
  - "open source project success"
  - "growing an open source project"
---

> Growth is the crucial aspect of a successful open source project. If you're at this stage in your open source maintainer journey, here are some things to consider when thinking of growth strategies.

Reviewer 2: I think we can omit this first paragraph and start on line 16.

## When to grow your project

> When it comes to deciding on whether your open source project is ready to grow, consider reviewing your project's [Repository Insights page](../features/repo-insights.md) and asking yourself the following questions:

Reviewer 2: I'm not sure it makes sense to mention repo insights pages right here because it doesn't have to do with the questions you're asking. 

I would also avoid mentioning stars and forks as a goal for a project. Instead, you can ask "Are there active community members who would be interested in progressing to an official role?

- Are there tasks that are becoming too overwhelming for me and the team?
- How can I improve and maintain my project's current and pending features?
- How many stars and forks do I want to reach this time, and who can help me achieve it?

If you have said yes to these questions, it's time to develop your growth strategies. If you haven't already, consider creating different teams for specific purposes. For example, you can create a docs team for documentation maintenance, a triage team for reviewing and approving issues, or a maintainer team to help you with overall project oversight. Once you have decided the types of teams you want to add to your project, it's time to start building the open source team of your dreams.

## How to identify new contributors and team members

> When it comes to planning the growth strategies for your open source project, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important. When doing this process, consider going to the "Contributors" tab in your project's Repository Insight Page. While you do this, ask yourself the following questions:

Reviewer 2: Let's make this a little clearer:

 > When it comes to planning the growth strategies for your open source project, picking the right people will determine how successful your project will be. You can discover the right people for your project by starting with a [Repository Insight Page](add link to feature). Once you've created a Repository Insight Page, you can use the Contributors dashboard to help you get a better understanding of your existing contributors.

> 

Reviewer 2: I don't think we need this. The next line should say something like:

> If you have a reliable group of contributors, adding them to a [Contributor Insight Page](add link to feature docs) can help you to better understand their contributions to your project and the open source ecosystem.

- How often do they contribute to yours and other open source projects in general?
- Are they invested in your project's success?

Once you have made your observations, [create a Contributor Insight Page for your project](../maintainers/maintainers-guide.md#creating-a-new-contributor-insight-page). This will help you to make decisions about what steps you want to take next. Here are some approaches you can use to help you:

- Monitor contributors to see if they are active enough in your repositories to contribute as a team member.
- Identify your top contributors to extend an invitation to join a team.
- Identify active contributors in other repositories who may be interested in joining your team.

If you need more tips on how to pick people that can help grow your project, check out our blog post, ["How to Build Your Open Source Dream Team"](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90).

> ## Recruiting team members and contributors

Reviewer 2: I think this might be more effective if we present it in a different way. 

### Recruiting Team Members and Contributors Outside of Your Project

If you've looked at your Repository Insight Page and realize there are no contributors that could help you, you can consider recruiting new contributors. To do that, consider creating a...

When you're ready to recruit your team members, consider creating a [Repository Insight Page](../features/repo-insights.md) with projects that are similar to yours. From there, determine which contributors suit your project, add them to a list, and monitor their progress.  

:::tip

Consider checking out our [Understanding Repository Insights Data](../features/repo-insights.md) guide to give you an idea on what to look for when narrowing down your list.

:::

> Once you've picked the people you want to join your project, recruit them. You can do this by going to their [OpenSauced profiles](../contributors/contributors-guide.md#creating-your-profile) and sending them a message via the contact information they provide.

Reviewer 2: Let's remove the profile section. You can end with a simple sentence like, "Once you've identified the people you want to ask, reach out to them, providing a clear message and a list of expectations.

## Understanding team participation

Once you have recruited your official team members, consider adding them to your project's Contributors Insights page by using [the team sync feature](../features/contributor-insights.md#how-to-create-a-contributor-insight-page). This can be useful in monitoring their progress.

> 

Reviewer 2: You have to create a team in GH to be able to use the team sync feature.

I like this heading and I think it will be a helpful paragraph, but I'd love to see you develop it more. _How_ do you understand team participation? What are the things maintainers should be considering? What happens if they see the participation is minimal?

:::

Once you've identified the people you want to ask, reach out to them, providing a clear message and a list of expectations.

After the people from your list has agreed to join your project, create [your team on GitHub](https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team) and [sync it](../welcome/faqs.md#sync-your-github-team) to your project's Repository Insight page.

![team sync gif](../../static/gif/team-sync.gif)

## Understanding team participation

To ensure that your open source project reaches it growth strategy goal(s), it is important that your team members actively participate in the project as much as possible. The [Contributors tab](../maintainers/maintainers-guide.md#contributors) in your project's Repository Insights page can be helpful in fostering participation. Here are some strategies you can use this tab to monitor your team member's progress:

- Identify the amount of repositories they contributed to in the Repositories section to see if they are immersed in the project.
- Take note of the date of their recent contribution in the Last Contributed section to see if it aligns with the number in the Repositories section.

> - Check the level in the Act tab to determine how often they contribute to the project

Reviewer: - What is "Act"?
- It needs a period at the end of the sentence.
- Inconsistency. You use  the term "section" in the other bullet points, and "tab" here.

:::tip

If you see "Low" in the Act tab, consider messaging your team member about this information and request a meeting to discuss possible strategies to help them progress.

:::

## Creating Transparency and Communication With Your Team

> The methods  utilized to collaborate and communicate with your team play a crucial role in the growth of your open source project.  Creating and sharing your project's [workspace](../features/workspaces.md) page with your team members can be helpful in fostering a collaborative environment. After creating this page, here are some ways to leverage its information to engage with your team:

Reviewer: ```suggestion
The methods utilized to collaborate and communicate with your team play a crucial role in the growth of your open source project. Creating and sharing your project's [Workspace](../features/workspaces.md) page with your team members can be helpful in fostering a collaborative environment. After creating this page, here are some ways to leverage its information to engage with your team:
```

> - Analyze the closed number Pull Requests and Issues tabs and the PR velocity to think of new ways to quickly review and merge people's contributions.

Reviewer: I don't quite understand what do you mean here. Can you help me to understand it better?

Author: I was referring to the dashboard in the Workspace page(refer to the red symbol in the Engagement tab in the screenshot below)
![screenshot of engagement tab](https://github.com/open-sauced/docs/assets/105683440/79cee87e-8565-420e-bd65-c099e610869f)

Reviewer: Ah, I see. You're looking at those.

However, you might want to formulate the wording better. I'm not sure about native-English speakers, but I certainly don't understand this at all. 😅

> - Use the amount of stars and forks displayed in the Engagement tab to discuss the effectiveness of current promotion strategies and to determine new methods to promote the project.

Reviewer: There is no Engagement "tab".

Author: Yes there is @adiati98 

![screenshot showing Engagement tab](https://github.com/open-sauced/docs/assets/105683440/e57855ed-f780-435a-9050-cbfe6311a27b)

Reviewer: I know there's an "Engagement", but I'm not sure if it calls a "tab".

Author: Ah so that's the name! I think statistic card would be a good name.

Reviewer: Yep. Also, it would be great if you try to search around first, then ask for clarity with some explanation of what you did and you've get so far when you're not sure about something rather than assuming. 🙂

- Are there tasks that are becoming too overwhelming for me and/or team?
- How can I improve and maintaining my project's current and pending features?
- How many stars and forks do I want to reach this time, and who can help me achieve it?
If you have said yes to these questions, it's time to develop your growth strategies. If you haven't already, consider creating different teams for specific purposes. For example, you can create a docs team for documentation maintenance, a triage team for reviewing and approving issues, or a maintainer team to help you with overall oversight of the project.  Once you have decided the types of teams you want to add to your project, you can create separate pages for each team in your project's Workspace page. Check out the [Creating a New Repository Insights Page](../docs/maintainers-guide.md#adding-repositories-to-your-workspace).

### How to identify new contributors and team members

> Like Superman, Wonder Woman, the Flash, and Batman in ***The Justice League***, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important.  When doing this process, consider going to their OpenSauced profiles and checking their [dev card](../docs/features/dev-card.md). While you do this, ask yourself the following questions:

Reviewer 2: We have to be careful here. Pop culture references are not accessible to everyone, and for our docs we want to make sure that a universal audience can understand and relate to them.

Reviewer: I think, unlike blog posts, docs should be something "semi-formal" if not entirely "formal". Because docs is mainly talking about products/projects, and how to interact with them. Honestly, I've never seen docs written in an informal and non-universal way anywhere. My recommendation is to write this down in "semi-formal" style like what we have now.

Reviewer 2: You'll notice in that post that I don't reference specific super heroes. I'm going to push back against the idea that batman and superman are universal. 

The concept of superheroes is always more universal than specifying them by name; however, there's a difference in audience and tone with docs. I consider our docs to be semi-formal and blog posts can be more conversational, because the audience likely varies from week to week.

Docs should be consistent and relevant at all times and for the broadest audience. So we need to be very deliberate here with the style and continue to maintain the style that we've established throughout the docs.

- How often do they contribute to yours and other open source projects in general?
- Are they invested in your project's success?

> 

Reviewer 2: We have to be specific as to whether we want them to create a repository insight page or a contributor insight page.

Author: Hey @BekahHW. Here, I decided that maintainers could use a contributor insight page. I got the inspiration from [this part of the maintainers guide](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#why-create-a-contributor-insight-page).

- How have they grown since you found their profile?
- What types of contributions does this person frequently make to yours and other open source projects?
From there, consider narrowing down your list to If you need more tips on how to pick people that can help grow your project, check out our blog post, ["How to Build Your Open Source Dream Team"](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90).

### Recruiting team members and contributors

When you're ready to recruit your team members, consider creating an Insights page with projects that are similar to yours. From there, determine which contributors you think suits your project, add them to a list, and monitor their progress.  Once you've finished your observations, recruit them. You can do this by going to their OpenSauced profiles and sending them a message via the contact information they provide.

> :::note If both of you have a PRO account, you can also send them invites using the [connections feature](../../docs/features-connections.md).

Reviewer 2: This is not true

Reviewer: We omitted connection from [our docs](https://docs.opensauced.pizza/). So, I don't think we should even mention connections anywhere in the docs.
